dark/fasta.py

```diff
@@ -79,13 +79,17 @@ class FastaReads(Reads):
     """
     Subclass of L{dark.reads.Reads} providing access to FASTA reads.
 
-    @param file_: A C{str} file name or file handle, containing
+    @param _file: A C{str} file name or file handle, containing
         sequences in FASTA format,
     @param readClass: The class of read that should be yielded by iter.
+    @param checkAlphabet: An C{int} or C{None}. If C{None}, alphabet checking
+        will be done on all reads. If an C{int}, only that many reads will be
+        checked. (Pass zero to have no checks done.)
     """
-    def __init__(self, file_, readClass=DNARead):
-        self.file_ = file_
-        self.readClass = readClass
+    def __init__(self, _file, readClass=DNARead, checkAlphabet=None):
```

Comment:
Can you not say `checkAlphabeth=False`? Because on line 103 you do `count < checkAlphabet` and I find it unclear if `checkAlphabeth` could be `None`.